docs/JLBP-12.md

```diff
@@ -1,19 +1,25 @@
 # [JLBP-12] Make level of support and API stability clear
 
-- Clearly document the lifecycle status (and corresponding level of support)
-  of the library. Examples include:
-  - Incubating (no support expected)
-  - Stable (SLOs in place to handle issues)
-  - Maintenance (only bugs will be addressed)
-  - Deprecated / Abandoned / Inactive (no work is expected)
-- Clearly state the API and behavioral stability clients can expect from the
-  library surface. Example statements:
+Document the lifecycle status and corresponding level of support
+of the library. Examples include:
+
+  - Incubating: no support expected
```

Comment:
```suggestion
- Incubating: no support expected
```

I don't think you want this indent for this list.

Reply:
done